Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java

```diff
@@ -271,6 +269,10 @@ public void onClick(View view) {
                         .setDefaultURL("https://play.google.com/store/apps/details?id=com.kindred.android")
                         .setCallback(new Branch.BranchLinkShareListener() {
                             @Override
+                            public void onShareLinkDialogLaunched() {
```

Comment:
Partners want to add a progress in the window between calling showShareDialog and the time at which dialog actually appears. This may take a second or 2  in some devices depending on the preferred option setting